product_pricelist_fixed_price/README.rst

```diff
@@ -0,0 +1,26 @@
+Fixed price on pricelist rule
+=============================
+Adds a new option on pricelist rules to set a fixed price. This is made using
+a trick that writes on the back 100% in the discount to the base price to get
+a zero base that will add only the price we put in the surcharge price.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
```

Comment:
Here there should be more people (you for example :wink:, and @hbrunn, if I don't remember bad).